webui/react/src/pages/F_ExpList/glide-table/GlideTable.tsx

```diff
@@ -224,15 +235,36 @@ export const GlideTable: React.FC<Props> = ({
     [data, columnIds, columnDefs],
   );
 
-  const handleGridSelectionChange = useCallback((newSelection: GridSelection) => {
-    const [, row] = newSelection.current?.cell ?? [undefined, undefined];
+  const handleGridSelectionChange = useCallback((cell: Item) => {
+    const [col, row] = cell;
+    if (col !== 0) return;
```

Comment:
the expected behavior is for clicking on a row to select the experiment, so we should probably try avoid this.

if the goal is to avoid selecting a row when we click outside the context menu, we can do [this](https://github.com/mapmeld/determined/pull/2)

Reply:
The issue that I had was a right-click to *open* the context menu selected the row. I can probably work with this